include/jemalloc/internal/jemalloc_internal_inlines_d.h

```diff
@@ -0,0 +1,220 @@
+#ifndef JEMALLOC_INTERNAL_INLINES_D_H
```

Comment:
Cannot move them elsewhere because of the interdependency with prof_inlines.h

Reply:
Is `prof_sample_aligned` the only part needed from `prof_inlines.h`? If so let's take that out of the prof header, and probably rename to `nonfast_aligned` or `special_aligned`. Then you don't have to add yet another header?

Reply:
Yes, since `prof_sample_aligned` is merely one line. I copied it and named it as `nonfast_aligned`. With this I can put everything in jemalloc_internal_inlines_c.h instead of creating a new file. I kept the original function in prof_inlines.h since it is needed there by other functions there as well.